apps/web/src/components/Router.tsx

```diff
@@ -210,6 +210,13 @@ const createRoute = (locale: Locales) =>
                       "../pages/ProfileAndApplicationsPage/ProfileAndApplicationsPage"
                     ),
                 },
+                // {
+                //   path: "dashboard-test",
+                //   lazy: () =>
```

Comment:
I think this will only be needed for test purposes, the page replaces an existing route later. I added the hyphen as `/applicant/dashboard` is an existing nginx redirect

Reply:
Ya that should work for now :+1: